editor/plugins/script_text_editor.cpp

```diff
@@ -209,7 +290,12 @@ void ScriptTextEditor::_notification(int p_what) {
 	if (p_what == NOTIFICATION_READY) {
 
 		//emit_signal("name_changed");
+		_load_theme_settings();
 	}
+	// if (p_what == EDITOR_SETTINGS_CHANGED) {
+
+	// 	_load_theme_settings();
+	// }
```

Comment:
Not to sure @akien-mga will agree with this :P

Reply:
forgot about it.